tunnel/tunnel.go

```diff
@@ -607,7 +612,12 @@ func buildSSHChannels(serverName, channelType string, source, destination []stri
 func getForward(channelType, serverName string) (*ForwardConfig, error) {
 	var f *ForwardConfig
 
-	cfg, err := NewSSHConfigFile()
+	cfgPath, err := NewDefaultSSHConfigFilePath()
+	if err != nil {
+		return nil, fmt.Errorf("error building ssh configuration file path: %v", err)
+	}
+
+	cfg, err := NewSSHConfigFile(cfgPath)
```

Comment:
This is not correct because it will always use the default home dir and the user might be passed a custom location through the `config` flag.